core/src/components/img/test/draggable/img.e2e.ts

```diff
@@ -0,0 +1,17 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('img: rendering', () => {
```

Comment:
~ Applies to each `img.e2e.ts` file, should these be updated to be:
`img: draggable`
`img: basic`

etc.?